examples/11_stream_k_gemm/CMakeLists.txt

```diff
@@ -1,25 +1,6 @@
 set(TARGET stream_k_gemm)
 
-set(XETLA_KERNEL_FLAGS ${XETLA_KERNEL_FLAGS} -fsycl)
```

Comment:
why only update this example?

Reply:
The cmake options of this example are the same as the cmake options in the project.   is a duplicate code